<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Deste Sayısı :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>252</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Destedeki Ürün Sayısını Girin :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>110</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>60</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>168</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ürün Sayısını Girin :</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>214</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Kutunun Boyutları</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>140</y>
      <width>214</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Destenin Boyutları</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>220</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>80</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Cambria</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>En</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <family>Cambria</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Genişlik</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <family>Cambria</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Yükseklik</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="textEdit_5">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>260</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_6">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>260</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="hesaplaButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>310</y>
      <width>144</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Hesapla</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>180</y>
      <width>80</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="font">
        <font>
         <family>Cambria</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>En</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_11">
       <property name="font">
        <font>
         <family>Cambria</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Genişlik</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_12">
       <property name="font">
        <font>
         <family>Cambria</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Yükseklik</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="textEdit_7">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>220</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_8">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>180</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>370</y>
      <width>501</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
